remote/bitbucketserver/bitbucketserver.go

```diff
@@ -240,12 +263,16 @@ func (*client) Status(*model.User, *model.Repo, *model.Build, string) error {
 }
 
 func (c *client) Netrc(user *model.User, r *model.Repo) (*model.Netrc, error) {
-	u, err := url.Parse(c.URL) // TODO strip port from url
+	u, err := url.Parse(c.URL)
+	//remove the port
```

Comment:
if there is an error here, calling `u.Host` below will result in a nil pointer and panic